observability/distr_tracing/distr-tracing-tempo-configuring.adoc

```diff
@@ -1,38 +1,43 @@
+= Configuring the {DTShortName}
 :context: distr-tracing-tempo-configuring
 
 toc::[]
 
+[TIP]
+====
+For information about configuring the deprecated {JaegerShortName}, see xref:../../observability/distr_tracing/distr_tracing_jaeger/distr-tracing-jaeger-configuring.adoc#distr-tracing-jaeger-configuring[Configuring] in the {JaegerShortName} documentation.
+====
+
 The {TempoOperator} uses a custom resource definition (CRD) file that defines the architecture and configuration settings for creating and deploying the {TempoShortName} resources. You can install the default configuration or modify the file.
 
 ifdef::openshift-enterprise,openshift-dedicated[]
 [id="configuring-storages_{context}"]
 == Configuring back-end storage
 
-For information about configuring the back-end storage, see xref:../../../storage/understanding-persistent-storage.adoc#understanding-persistent-storage[Understanding persistent storage] and the relevant configuration section for your chosen storage option.
+For information about configuring the back-end storage, see xref:../../storage/understanding-persistent-storage.adoc#understanding-persistent-storage[Understanding persistent storage] and the relevant configuration section for your chosen storage option.
 endif::[]
 
 include::modules/distr-tracing-tempo-config-default.adoc[leveloffset=+1]
 
 [role="_additional-resources"]
 .Additional resources
-* xref:../../../observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-installing.adoc#dist-tracing-tempo-installing[Installing a TempoStack instance]
-* xref:../../../observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-installing.adoc#dist-tracing-tempo-installing[Installing a TempoMonolithic instance]
+* xref:../../observability/distr_tracing/distr-tracing-tempo-installing.adoc#distr-tracing-tempo-installing[Installing a TempoStack instance]
+* xref:../../observability/distr_tracing/distr-tracing-tempo-installing.adoc#distr-tracing-tempo-installing[Installing a TempoMonolithic instance]
 
 include::modules/distr-tracing-tempo-config-query-frontend.adoc[leveloffset=+1]
 
 [role="_additional-resources"]
 .Additional resources
-* xref:../../../nodes/scheduling/nodes-scheduler-taints-tolerations.adoc#nodes-scheduler-taints-tolerations-about_nodes-scheduler-taints-tolerations[Understanding taints and tolerations]
+* xref:../../nodes/scheduling/nodes-scheduler-taints-tolerations.adoc#nodes-scheduler-taints-tolerations-about_nodes-scheduler-taints-tolerations[Understanding taints and tolerations]
 
 include::modules/distr-tracing-tempo-config-spanmetrics.adoc[leveloffset=+1]
 
 [role="_additional-resources"]
 .Additional resources
-* xref:../../../observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-installing.adoc#dist-tracing-tempo-installing[Configuring the permissions and tenants]
+* xref:../../observability/distr_tracing/distr-tracing-tempo-installing.adoc#distr-tracing-tempo-installing[Configuring the permissions and tenants]
```

Comment:
This one links to "Installing the Distributed Tracing Platform".

Reply:
Same situation as in https://github.com/openshift/openshift-docs/pull/93472#discussion_r2189321709.